modules/build/src/main/scala/scala/build/Build.scala

```diff
@@ -256,9 +256,34 @@ object Build {
     val (testBuild, extraTestBuilds) =
       value(buildScope(Scope.Test, Some(mainBuild), Some(extraBuilds)))
 
+    copyResourceToClassesDir(mainBuild)
+    copyResourceToClassesDir(testBuild)
+
     Builds(Seq(mainBuild, testBuild), Seq(extraBuilds, extraTestBuilds))
   }
 
+  private def copyResourceToClassesDir(build: Build) = build match {
+    case b: Build.Successful =>
+      for {
+        resourceDirPath  <- b.sources.resourceDirs
+        resourceFilePath <- os.walk(resourceDirPath)
+        if os.isFile(resourceFilePath)
+        relativeResourcePath = resourceFilePath.relativeTo(resourceDirPath)
+        if !relativeResourcePath.toString.startsWith(
+          ".scala"
+        ) // dismiss files generated by scala-cli
+      } yield {
```

Comment:
No need of the yield here, as this method returns a `Unit` IIUC.
```suggestion
      } {
```